offload/CMakeLists.txt

```diff
@@ -35,11 +42,12 @@ endif()
 # TODO: Leftover from the move, could probably be just LLVM_LIBDIR_SUFFIX everywhere.
 set(OFFLOAD_INSTALL_LIBDIR "lib${LLVM_LIBDIR_SUFFIX}")
 
-set(LLVM_COMMON_CMAKE_UTILS ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
+set(LLVM_COMMON_CMAKE_UTILS ${CMAKE_CURRENT_SOURCE_DIR}/../cmake)
```

Comment:
This doesn't make sense to me, the whole point of a standalone build is that it's separate from LLVM, which would mean that this is the top level directory in the tree. Why are we just assuming that the LLVM modules directory is there?

I don't think putting this module stuff is necessary since we can just make the users pass in `OMPTARGET_OMPT_ENABLED` or something.

Reply:
I think it's ok to assume the openmp sources are still inside the llvm-project, right? It is standalone in the sense that it will not be built during the same build as llvm. This change was more so a correction on the current offload patch `${CMAKE_CURRENT_SOURCE_DIR}/cmake` == `offload/cmake`, which is not what `LLVM_COMMON_CMAKE_UTILS` is supposed to reference.

I was just keeping the autodetection in for OMPT as the system configuration still needs checked to verify whether it can be supported, regardless if the user passes it in or not.